java/src/jmri/jmrit/throttle/ThrottleWindow.java

```diff
@@ -249,9 +255,12 @@ private void initializeToolbar() {
         throttleToolBar.add(jbThrottleList);
 
         // Receptacle for Jynstruments
-        new URIDrop(throttleToolBar, files -> {
-            for (java.net.URI file : files) {
-                ynstrument(file.getPath());
+        new URIDrop(throttleToolBar, new Listener() {
+            @Override
+            public void URIsDropped(URI[] uris) {
+                for (URI uri : uris ) {
+                    ynstrument(new File(uri).getPath());
+                }
             }
         });
```

Comment:
Thanks for the merge of master.

In this particular case, it would be good to use the new syntax, since it makes the code easier to read:
```
        new URIDrop(throttleToolBar, uris -> {
            for (URI uri : uris) {
                ynstrument(new File(uri).getPath());
            }
        });
```